module3/lessons/Week2/FindingYourStrengths.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: page
+title: Finding Your Strengths
+---
+
+## Learning Goals
+* Understand what a strength is and what it can look like in practice
+* Identify challenges with using your strengths
+* Plan for how to discuss strengths and working preferences
+
+## Pre-Work
+
+Before we dive into a discussion of strengths, we are going to start to identify what our current strengths are.  
+
+<section class='call-to-action' markdown='1'>
+Spend 20 - 30 minutes taking the free [High 5 Strengths Assessment](https://test.high5test.com/registration).  Take a few moments to read through the descriptions of your top 5 strengths; we will revisit these later!
+</section>
+
```

Comment:
There is a lot of sit and get in this lesson, trying to think of how to make it a touch more student interaction. What if we start with breakout rooms of 3/4 and ask students to discuss why they think it's important to identify your strengths?

Reply:
Great idea!

Reply:
I'm going to include this activity as an instructor note - I think if they see the prompt ahead of time, they might do some pre-research.